src/main/java/arb/ui/project/ProjectCard.java

```diff
@@ -73,6 +74,9 @@ public ProjectCard(Project project, int displayedIndex) {
         }
 
         status.setText("Status: " + (project.isOverdue() ? "OVERDUE" : project.getStatus().toString()));
+        if (project.isOverdue()) {
+            status.setTextFill(Color.RED);
```

Comment:
This line needs to be changed to `status.setStyle("-fx-text-fill: red");` for this to work.